src/commands/prompt.ts

```diff
@@ -16,24 +16,36 @@ const promptModule: CommandType = {
     ),
   execute: async (interaction: CommandInteraction<CacheType>) => {
 
-    let channelId = interaction.guild?.channels.cache.find(channel => channel.name === Config?.prompt.channel) as TextChannel
+    let configChannel = Config?.prompt.channel
+    if (configChannel && configChannel?.length === 0) {
+      await interaction.reply({ content: "We are missing channel name in config file" })
+      
+    } else {
+      let channelId = interaction.guild?.channels.cache.find(channel => channel.name === configChannel) as TextChannel
+
+      if(channelId === undefined){
+        await interaction.reply({ content: `Channel ${inlineCode(configChannel!.toString())} does not exist in this guild`})
+        
```

Comment:
@kilbouri I'm not sure but is this what you meant?

Reply:
The first check, the length one, isn't necessary. But yes, responding with a proper error is what I meant.